scapy/contrib/roce.py

```diff
@@ -228,6 +272,30 @@ class AETH(Packet):
     ]
 
 
+class DETH(Packet):
+    name = "DETH"
+    fields_desc = [
+        XIntField("qkey", 0),
+        XByteField("reserved", 0),
+        XBitField("sqp", 0, 24)
+    ]
+
+    def guess_payload_class(self, payload):
+        # type: (bytes) -> Type[Packet]
+        bth = self.underlayer
+        if isinstance(bth, BTH):
+            if bth.opcode == opcode('UD', 'SEND_ONLY_WITH_IMMEDIATE')[0]:
+                return Immdt
+        return Packet.guess_payload_class(self, payload)
+
+
+class Immdt(Packet):
+    name = "Immdt"
```

Comment:
Is there a full name to this packet? What does Immdt means? Would be cool to add it in `name`

Reply:
Yes, the full name in the spec is "ImmDt - Immediate Data Extended Transport Header." I'll fix that. By the way, where does scapy use `name`?

Reply:
Mostly in `show()`